src/components/tipsslider/tipsslider.json

```diff
@@ -0,0 +1,50 @@
+[
```

Comment:
It might be helpful to have the json in the format that microworlds use as a sample – i.e.  carousel items chunked into sets.